starknet_py/net/account/account_client.py

```diff
@@ -318,9 +318,12 @@ async def _get_max_fee(
                 "Max_fee and auto_estimate are exclusive and cannot be provided at the same time."
             )
 
-        if isinstance(transaction, Declare) and transaction.version != 1:
+        if (
+            isinstance(transaction, (Declare, DeployAccount))
+            and transaction.version != 1
+        ):
             raise ValueError(
-                "Estimating fee for Declare transactions with versions other than 1 is not supported."
+                "Estimating fee for Declare/DeployAccount transactions with versions other than 1 is not supported."
```

Comment:
I wonder if we should be raising more specific errors per transaction type? But perhaps it's not needed.

Reply:
I think it is not needed. It is the same error

Reply:
I can do it this way:
`f"Estimating fee for {type(transaction)} transactions..."`

Reply:
Okay, let's merge it then.